internal/core/api.rs

```diff
@@ -81,21 +81,21 @@ impl PhysicalPosition {
     }
 }
 
-/// The requested position to render the window at.
+/// The position of the window in either physical or logical pixels.
```

Comment:
unrelated to this path, but i think it would be nice to reference ``[`Window::set_position`]``

Reply:
Good idea, let me try that